.github/workflows/release.yml

```diff
@@ -83,6 +83,26 @@ jobs:
           tag: ${{ github.ref }}
           overwrite: true
   changelog:
+    exclude:
+      labels:
+        - no-changelog
+      authors:
+        - dependabot
+        - pre-commit-ci
+    categories:
+      - title: Breaking Changes 🛠
+        labels:
+          - breaking-change
+      - title: Enhancements 🎉
+        labels:
+          - enhancement
+      - title: Bug Fixes 🐛
+        labels:
+          - bug-fix
+      - title: Other Changes
+        labels:
+          - "*"
```

Comment:
What is this for? I can't find same in leapp, I am just curios how it works

Reply:
this will make release notes into categories something like this
![image](https://github.com/oamg/infra.convert2rhel/assets/56552845/b5295eb5-292a-47b7-9d86-93bcc2f1418f)